### ISSUE TITLE
SystemError: opening file /.../homebrew-core/Formula/qt5.rb: No such file or directory

### ISSUE THREAD

**Issue by scls19fr** [barche/QML.jl#39]
I'm trying to build QML with JuliaPro 0.5.1.1 (under Mac OS 10.12.4 ) using

`Pkg.build("QML")`

and I get the following error message

```
===================================================================================[ ERROR: QML ]===================================================================================

LoadError: SystemError: opening file /Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qt5.rb: No such file or directory
while loading /Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/QML/deps/build.jl, in expression starting on line 36

====================================================================================================================================================================================

==================================================================================[ BUILD ERRORS ]==================================================================================

WARNING: QML had build errors.

 - packages with build errors remain installed in /Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5
 - build the package(s) and all dependencies with `Pkg.build("QML")`
 - build a single package by running its `deps/build.jl` script

=
```

Looking for `qt*` in `/Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/` directory

```
$ ls /Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qt*
/Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qt.rb
/Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qt@5.5.rb
/Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qt@5.7.rb
/Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qtads.rb
/Applications/JuliaPro-0.5.1.1.app/Contents/Resources/pkgs-0.5.1.1/v0.5/Homebrew/deps/usr/Library/Taps/homebrew/homebrew-core/Formula/qtfaststart.rb
```

Any idea?

**Comment by scls19fr** [comment]
Qt was installed previously using

```
brew install qt5
```

I tried to install it using

```
brew install qt
==> Downloading https://homebrew.bintray.com/bottles/qt-5.8.0_2.sierra.bottle.tar.gz
######################################################################## 100,0%
==> Pouring qt-5.8.0_2.sierra.bottle.tar.gz
==> Caveats
We agreed to the Qt opensource license for you.
If this is unacceptable you should uninstall.

This formula is keg-only, which means it was not symlinked into /usr/local.

Qt 5 has CMake issues when linked

If you need to have this software first in your PATH run:
  echo 'export PATH="/usr/local/opt/qt/bin:$PATH"' >> ~/.bash_profile

For compilers to find this software you may need to set:
    LDFLAGS:  -L/usr/local/opt/qt/lib
    CPPFLAGS: -I/usr/local/opt/qt/include
For pkg-config to find this software you may need to set:
    PKG_CONFIG_PATH: /usr/local/opt/qt/lib/pkgconfig

==> Summary
🍺  /usr/local/Cellar/qt/5.8.0_2: 8,667 files, 246.2MB
```

and now it works fine.

Sorry for noise

**Comment by barche** [comment]
OK, this should also be fixed in the latest release by PR #38

**Comment by scls19fr** [comment]
Thanks @barche
